build.gradle.kts

```diff
@@ -280,9 +280,26 @@ subprojects {
 
 allprojects {
   plugins.withId("com.vanniktech.maven.publish.base") {
+    val artifactoryUrl = System.getProperty("artifactory_url")
```

Comment:
nit: for consistency with `publish_username` and `publish_password` and for more generality, maybe rename this to `publish_url`?